bofire/strategies/botorch/qehvi.py

```diff
@@ -27,17 +27,32 @@
     Objective,
     TargetObjective,
 )
-from bofire.strategies.botorch.base import BotorchBasicBoStrategy
+from bofire.strategies.botorch.base import BotorchBasicBoStrategy, add_exclude
 from bofire.utils.multiobjective import get_ref_point_mask
 from bofire.utils.torch_tools import get_output_constraints, tkwargs
 
 
 # TODO: unite this by using get_acquisiton
 class BoTorchQehviStrategy(BotorchBasicBoStrategy):
+
+    type: Literal["BoTorchQehviStrategy"] = "BoTorchQehviStrategy"
+
     ref_point: Optional[dict] = None
     ref_point_mask: Optional[np.ndarray] = None
     objective: Optional[MCMultiOutputObjective] = None
 
+    # TODO: unhide fields
+    def json(self, **kwargs):
+        return super().json(
+            **add_exclude(kwargs, "ref_point", "ref_point_mask", "objective")
```

Comment:
the ref point should not be excluded,

Reply:
yes, it is currently hidden to allow for correct serialization.
will be "fixed" later 8see the TODOs).